de-lib/src/main/java/org/iplantc/de/analysis/client/presenter/AnalysesPresenterImpl.java

```diff
@@ -136,6 +138,28 @@ public void onSuccess(String result) {
 
     }
 
+    private final class CompleteAnalysisServiceCallback extends AnalysisCallback<String> {
+        private final Analysis ae;
+
+        public CompleteAnalysisServiceCallback(final Analysis ae) {
+            this.ae = ae;
+        }
+
+        @Override
+        public void onFailure(Integer statusCode, Throwable caught) {
+            SafeHtml msg = SafeHtmlUtils.fromString(appearance.stopAnalysisError(ae.getName()));
```

Comment:
I decided this and the other string (which just refer to stopping the job, and don't use the word cancel or complete) were fine to reuse here.